apps/search-poc/src/containers/WorkspaceList.tsx

```diff
@@ -19,8 +22,9 @@ type WorkspaceConfig = {
 const WorkspaceListContainer: React.FunctionComponent<{
   workspaceConfig: WorkspaceConfig[];
   onWorkspaceSelected: (workspaceId: string) => void;
-  history: History;
-}> = ({ workspaceConfig, onWorkspaceSelected, history, children }) => {
+  handleClick: (params: HandleClickParams) => void;
+}> = ({ workspaceConfig, onWorkspaceSelected, handleClick }) => {
```

Comment:
children is left undefined after merge I guess

> ./src/containers/WorkspaceList.tsx

  > Line 73:  'children' is not defined  no-undef

Reply:
Fixed